utilities/scripts/New-DscResourceModule.ps1

```diff
@@ -0,0 +1,106 @@
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Low')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingPositionalParameters', '', Justification = 'Positional parameters are used for simplicity. Targeting PS 7+')]
+    param (
+        [Parameter(Mandatory)]
+        [string]$DscResourceModule,
+
+        [Parameter(Mandatory)]
+        [string]$Description,
+
+        [Parameter()]
+        [string]$BasePath = (Join-Path $PSScriptRoot '..' '..'),
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string[]] $DscResourceToExport
+
+    )
+
+    $resourcePath = Join-Path $BasePath 'resources' $DscResourceModule
+    $testsPath = Join-Path $BasePath 'tests' $DscResourceModule
+
+    # Create directories if they do not exist
+    if (-not (Test-Path -Path $resourcePath)) {
+        Write-Verbose -Message "Creating directory: $resourcePath"
+        $null = New-Item -ItemType Directory -Path $resourcePath -Force
+    }
+
+    if (-not (Test-Path -Path $testsPath)) {
+        Write-Verbose -Message "Creating test directory: $testsPath"
+        $null = New-Item -ItemType Directory -Path $testsPath -Force
+    }
+
+    $moduleManifestPath = (Join-Path $BasePath 'resources' $DscResourceModule "$DscResourceModule.psd1")
+
+    $moduleManifestParams = @{
+        Path              = $moduleManifestPath
+        RootModule        = "$DscResourceModule.psm1"
+        ModuleVersion     = '0.1.0'
+        Author            = 'Microsoft Corporation'
+        CompanyName       = 'Microsoft Corporation'
+        Copyright         = '(c) Microsoft Corporation. All rights reserved.'
+        Description       = $Description
+        PowerShellVersion = '7.2'
+        FunctionsToExport = @('test')
+        CmdletsToExport   = @()
+        VariablesToExport = @()
+        AliasesToExport   = @()
+    }
+
+    if ($DscResourceToExport) {
+        # New module manifest does not properly handle arrays of strings :(
+        $moduleManifestParams.Add('DscResourcesToExport', @($DscResourceToExport))
+    }
+
+    if (-not (Test-Path $moduleManifestPath)) {
+        if ($PSCmdlet.ShouldProcess($moduleManifestPath, 'Create module manifest')) {
+            Write-Verbose -Message ($moduleManifestParams | ConvertTo-Json -Depth 10 | Out-String)
+            New-ModuleManifest @moduleManifestParams
+        }
+
+        # Workaround for issue: https://github.com/PowerShell/PowerShell/issues/5922
+        $fileContent = Get-Content $moduleManifestPath
+        $newLicenseUri = "LicenseUri = 'https://github.com/microsoft/winget-dsc/blob/main/LICENSE'"
+        $fileContent = $fileContent -replace '# LicenseUri = ''''', $newLicenseUri
+        $newProjectUri = "ProjectUri = 'https://github.com/microsoft/winget-dsc'"
+        $fileContent = $fileContent -replace '# ProjectUri = ''''', $newProjectUri
+        $newPrerelease = "Prerelease = 'alpha'"
+        $fileContent = $fileContent -replace '# Prerelease = ''''', $newPrerelease
```

Comment:
Please see explicit  comment above: # Workaround for issue: https://github.com/PowerShell/PowerShell/issues/5922

Reply:
I saw the comment. You can include these parameters directly instead of putting it in a nested hashtable. `New-ModuleManifest` has explicit switches for these. You can see this if you run `New-ModuleManifest -Path <path> -LicenseURI 'https://github.com`